### PR TITLE
added Nclient.html

### PR DESCRIPTION
**Pull request by Raqkel**
added more code and please check my js.bin code you sent me to see if i added it right to my code since i dont think its working

### CODE REVIEW

File: Nclient.html [hunk 1]
```diff
@@ -23,56 +23,435 @@
...
+		$('.js-change-modal').on('click', function() {
+  			var _this = $(this);
+  			window.modalName = _this.attr('data-modal');
+	});
+		
+		$(document).ready(function(){
+    	$('.modal').modal();
+  	});
+		$(document).ready(function() {
+    	$('select').material_select();
```
Comment (eddiedotdev) [inline Nclient.html:453]: you dont want to have 2 `document.ready`'s here. Move `$('select').material_select();` into your first `document.ready` (under line `450`)

File: Nclient.html [hunk 2]
```diff
@@ -23,56 +23,435 @@
...
 
 
 <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/js/materialize.min.js"></script>
 
 	<script>
+
 		$(document).ready(function() {
     	$('select').material_select();
   	});
```
Comment (eddiedotdev) [inline Nclient.html:441]: remove lines `439-441` since you initialize this function below

File: Nclient.html [hunk 3]
```diff
@@ -23,56 +23,435 @@
...
+		window.modalName = '';
+
+		$('.js-change-modal').on('click', function() {
```
Comment (eddiedotdev) [inline Nclient.html:444]: this function needs to go inside your `document.ready`

File: Nclient.html (hunk 1 above)
Comment (Raqkel) [reply Nclient.html:453]: oh cool ok

File: Nclient.html [hunk 4]
```diff
@@ -23,56 +23,435 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (eddiedotdev) [inline Nclient.html:431]: there's too much spacing here. Remove lines `412-431`

File: Nclient.html [hunk 5]
```diff
@@ -23,56 +23,435 @@
...
+						</div>
+						<div class="input-field col s3">
+							<input type="text" name="def_zip">
+							<label for="def_zip">Zip Code</label>	
+						</div>
+					</div>
+
+					<div class="row">
+						<div class="col s3">
+							<a class="waves-effect waves-light btn lime accent-3">Add</a>
```
Comment (eddiedotdev) [inline Nclient.html:402]: change this from an `<a>` tag into a button, for example: `<button class="waves-effect waves-light btn lime accent-3">Add</button>`

File: Nclient.html [hunk 6]
```diff
@@ -23,56 +23,435 @@
...
+					</div>
+
+					<div class="row">
+						<div class="input-field col s3">
+							<input type="text" name="def_city">
+							<label for="def_city">City</label>
 						</div>
+						<div class="input-field col s2">
+							<select>
+      						<option value="" disabled selected></option>
```
Comment (eddiedotdev) [inline Nclient.html:340]: remove `disabled selected` here, as the default will always be the first option unless you change that with javascript

File: Nclient.html [hunk 7]
```diff
@@ -23,56 +23,435 @@
...
+						</div>
+					</div>
+
+					<div class="row">
+						<div class="input-field col s3">
+							<input type="text" name="def_city">
+							<label for="def_city">City</label>
 						</div>
+						<div class="input-field col s2">
+							<select>
```
Comment (eddiedotdev) [inline Nclient.html:339]: this is gonna need a `name=""` attribute

File: Nclient.html [hunk 8]
```diff
@@ -23,56 +23,435 @@
...
+			   		<label for="lawfirm">Lawfirm</label>
+				</div>
+					
+					</div>
+				</div>
+			</div>
+			
+
+
+		<a class="waves-effect waves-light btn modal-trigger center" href="#modal1"></a>
```
Comment (eddiedotdev) [inline Nclient.html:69]: there's no text inside this `a` tag. You should remove this if you're not using it

**Comment by eddiedotdev** [comment]
The last thing I saw is that the alignment is all wonky on this page. Install the [HTMLBeautify](https://packagecontrol.io/packages/HTMLBeautify), run that before your next commit

**Comment by eddiedotdev** [comment]
change the name of the form from `NClient.html`, to `new-client.html`, also we gotta talk about how you structure your css file because you don't necessarily need to have separate css files for each page.

File: Nclient.html [hunk 9]
```diff
@@ -23,56 +23,435 @@
...
+		      						<option value="42">TN</option>
+		      						<option value="43">TX</option>
+		      						<option value="44">UT</option>
+		      						<option value="45">VT</option>
+		      						<option value="46">VA</option>
+		      						<option value="47">WA</option>
+		      						<option value="48">WV</option>
+		      						<option value="49">WI</option>
+		      						<option value="50">WY</option>
+		    						</select>
```
Comment (eddiedotdev) [inline Nclient.html:151]: unless Dustin specified he wants numbered values for states (not sure why he would do that), but change each value to equal the text, for example: `<option value="AZ">AZ</option>`

Comment (Raqkel) [reply Nclient.html:151]: oh no its something i just pulled from materialized , i didnt know what should be in there honestly so i kept it the same way as the example

Comment (Raqkel) [reply Nclient.html:151]: cool cool good to know tho

File: Nclient.html (hunk 3 above)
Comment (Raqkel) [reply Nclient.html:444]: um this im not sure about when you says to put it inside the document.ready. like this.....
$(document).ready window.modalName= ' ';
$( ' .js-change-modal ' ) .on ( ' click ', function () {

Comment (eddiedotdev) [reply Nclient.html:444]: so get rid of everything INSIDE your `<script>...</script>` tags and add this:

```
$(document).ready(function(){
  $('.modal').modal();
  $('select').material_select();

  // to switch the modal info
  window.modalName = '';
  $('.js-change-modal').on('click', function() {
    var _this = $(this);
    window.modalName = _this.attr('data-modal');
  });
  
});

```

Comment (eddiedotdev) [reply Nclient.html:444]: You have 1 `$(document).ready` function, and inside of that you initialize your jquery functions and run any code you want to fire onload of the page

**Pull request closed without merging**